config.json

```diff
@@ -258,8 +258,8 @@
         "entry_api",
         "filter",
         "match",
-        "mutablity",
-        "result"
+        "mutability",
+        "result_type"
```

Comment:
What's the motivation for renaming `result` to `result_type`?

Reply:
1) Because "results" looks weird,
2) especially since everyone talks about "Result" as in the Result type,
3) technically it's Result<T, Err>,
4) and possibly most important of all, Mr. Wikipedia mentions it: https://en.wikipedia.org/wiki/Result_type